Dockerfile

```diff
@@ -18,7 +18,7 @@ WORKDIR "${HOME}"
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
-        git-core \
+        git-core openssh-client \
```

Comment:
Can you please insert the following line between the `update` and the `install` one:
```bash
    && apt-get install --yes --only-upgrade openssl ca-certificates \
```